worker/src/RTC/PipeConsumer.cpp

```diff
@@ -573,9 +573,7 @@ namespace RTC
 				}
 			}
 
-			// Create a RtpStreamSend for sending a single media stream.
-			size_t bufferSize = params.useNack ? 600u : 0u;
-			auto* rtpStream   = new RTC::RtpStreamSend(this, params, bufferSize);
+			auto* rtpStream = new RTC::RtpStreamSend(this, params, this->rtpParameters.mid);
```

Comment:
Why are we passing MID here? are we taking into consideration that MID could be empty (not given, not used)?

Reply:
Ok, I understand it now. Resolved.

Reply:
> Why are we passing MID here

For retransmissions.

> are we taking into consideration that MID could be empty (not given, not used)?

Yes